client/src/components/DataFiles/fixtures/DataFiles.systems.fixture.js

```diff
@@ -40,7 +40,8 @@ const systemsFixture = {
         name: 'Shared Workspaces',
         scheme: 'projects',
         api: 'tapis',
-        icon: null
+        icon: null,
+        hideSearchBar: true
```

Comment:
This should be `false`, as by default, we do not want to hide the search bar
```suggestion
        hideSearchBar: false
```